asciiracer/resizer.py

```diff
@@ -0,0 +1,10 @@
+import os
+
+class Resizer(object):
+
+    def __init__(self):
+        self.current_size = tuple(int(i) for i in os.popen('stty size', 'r').read().split())
```

Comment:
Since this is just a one-liner, maybe it's better to have it as a function in the `misc.py`

Reply:
My initial idea was to be able to resize during gameplay with persisting the game state. This is the reason why it is separated.